src/iop/exposure.c

```diff
@@ -559,6 +575,23 @@ void gui_update(struct dt_iop_module_t *self)
   dt_bauhaus_slider_set_soft(g->black, p->black);
   dt_bauhaus_slider_set_soft(g->exposure, p->exposure);
 
+  g->spot_RGB[0] = 0.f;
+  g->spot_RGB[1] = 0.f;
+  g->spot_RGB[2] = 0.f;
+  g->spot_RGB[3] = 0.f;
+
+  // get the saved params
+  dt_iop_gui_enter_critical_section(self);
+
+  float lightness = 50.f;
```

Comment:
Maybe I'm missing something, but this does break the reset of the module. That is, even when clicking on reset on the module the lightness spot is not reset to 50%. Is that expected?

Reply:
The reset option applies only on module's params. These are not params but read/written from/to darktablerc, so the reset option does nothing here. The only way to reset is double click on sliders.

Reply:
Indeed ! Thanks.